gpflow/likelihoods/multilatent.py

```diff
@@ -120,3 +120,9 @@ def conditional_distribution(Fs) -> tfp.distributions.Distribution:
         super().__init__(
             latent_dim=2, conditional_distribution=conditional_distribution, **kwargs,
         )
+
+    def conditional_parameters(self, F):
+        return self._conditional_mean(F), self._conditional_variance(F)
+
+    def conditional_sample(self, F):
```

Comment:
Because this is quite an important method, would it be possible to add docstrings?

Reply:
Given that we are not converting all the likelihoods (rightfully so) I think it is important that we explain/document the code as good as possible so that people doing the work on other likelihoods know what to change/implement.